emctl/cmd/client/resource/types.go

```diff
@@ -69,20 +74,20 @@ const (
 type (
 	// VersionKind holds version and kind information for APIs
 	VersionKind struct {
-		APIVersion string `json:"apiVersion" jsonschema:"required"`
-		Kind       string `json:"kind" jsonschema:"required"`
+		APIVersion string `json:"apiVersion" yaml:"apiVersion" jsonschema:"required"`
+		Kind       string `json:"kind" yaml:"kind" jsonschema:"required"`
 	}
 
 	// MetaData is meta data for resources of the EaseMesh
 	MetaData struct {
-		Name   string            `json:"name" jsonschema:"required"`
-		Labels map[string]string `json:"labels" jsonschema:"omitempty"`
+		Name   string            `json:"name" yaml:"name" jsonschema:"required"`
+		Labels map[string]string `json:"labels" yaml:"labels" jsonschema:"omitempty"`
```

Comment:
`Label` is appeared in outputs of the `emctl get meshcontroller -o yaml` command even its value is null

Reply:
Updated